recipes/default.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Cookbook Name:: luigi
+# Recipe:: default
+#
+
+include_recipe 'python'
+
```

Comment:
Define an attribute `luigi.setup_user` and create one here if it is true.
(The same applies for group).